Sources/render/Makefile

```diff
@@ -0,0 +1 @@
+SRCS += $(SRCDIR)/render/font.cpp $(SRCDIR)/render/outputline.cpp $(SRCDIR)/render/renderer.cpp
```

Comment:
I think I'd prefer if `outputLine` lived in `debug` or similar instead of `render`.

Reply:
Thought about that as well, agreed. Would also opt for changing name of function+file as one could think "ah this is used for regularly displaying stuff" instead of "this is for debug purpose only". One should also evaluate if it could be completely optional.